experimental/ubuntu/Databricks Runtime 7/ubuntu18.04Python

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04 as builder
+
+RUN apt-get update && apt-get install --yes \
+    wget \
+    libdigest-sha-perl \
+    bzip2 \
+    strace
+
+# Download miniconda 4.5.12, then upgrade it to 4.6.12.
+RUN wget --quiet --output-document miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-4.5.12-Linux-x86_64.sh \
+    && (echo '866ae9dff53ad0874e1d1a60b1ad1ef8  miniconda.sh' | md5sum -c) \
+    && (echo 'e5e5b4cd2a918e0e96b395534222773f7241dc59d776db1b9f7fedfcb489157a  miniconda.sh' | shasum -a 256 -c) \
+    # Conda must be installed at /databricks/conda
+    && /bin/bash miniconda.sh -b -p /databricks/conda \
+    && rm miniconda.sh \
+    && /databricks/conda/bin/conda install --name base conda=4.6.12
+
+#This image required an update to start with the base of 18.04 created from containers/ubuntu/minimal this is attached as another dockerfile ubuntu18.04-minimal
+#Example from previous build: FROM databricksruntime/minimal:latest
+FROM nadroj09/test_jordan:minimal
```

Comment:
why is this base image hosted under your personal namespace?

Reply:
Looks like this PR is blocked/depends on PR #35 to create the `databricksruntime/minimal:latest` image, and the personal namespace is a placeholder for that.

Reply:
Correct, you need to update the databricksruntime/minimal:latest prior to creating this Dockerfile based on the reference.